services/src/main/java/org/jboss/windup/web/services/model/MigrationProject.java

```diff
@@ -57,9 +60,9 @@
 
     @Size(max = 4096)
     @NotNull
-    @Column(length = 4096, nullable = false)
+    @Column(length = 4096, nullable = false, unique = true)
     @org.hibernate.annotations.ColumnDefault("")
-    private String description;
+    private String description = "";
 
```

Comment:
Actually no, I intentionally put the FQCN there, so it's clear it is not standard JPA.